src/main/java/gui/utility/NavigationHelper.java

```diff
@@ -117,6 +119,9 @@ public static void enterPomodoroPage(MouseEvent event) throws IOException {
      * @throws IOException if the resource file cannot be found
      */
     public static void enterSettingsPage(MouseEvent event) throws IOException {
+        FXMLLoader loader = new FXMLLoader();
+        loader.setLocation(Objects.requireNonNull(NavigationHelper.class.getResource("/setting.fxml")));
```

Comment:
Does this setting page exist yet?

Reply:
it is not in this pr, but i think tahseen is still making adjustments to it